jnius/reflect.py

```diff
@@ -95,6 +95,12 @@ class Modifier(with_metaclass(MetaJavaClass, JavaClass)):
     isTransient = JavaStaticMethod('(I)Z')
     isVolatile = JavaStaticMethod('(I)Z')
 
+    @classmethod
+    def isPackageProtected(cls, modifier):
```

Comment:
i'm not sure about adding python code like this on a java class, i agree it's convenient, makes calls consistent with others, but it feels like a hack, as it's not a method of the java class, afaik. Will wait a bit to see if i come to term with the idea or not.

Reply:
When I added that I both thought it was clever and didn't like it because I'm making it different from what the actual Java `Modifier` class is like. This new method is only used in two places, so it doesn't actually add that much value anyway. I took it out.

Reply:
yeah, it's not a big code duplication anyway, i think it's better like this, thanks.